apps/vaporgui/TFWidget.cpp

```diff
@@ -1039,6 +1040,22 @@ string TFWidget::getTFVariableName(bool mainTF=true) {
 	return varname;
 }
 
+int TFWidget::convertOpacityToSliderValue(float opacity) const
+{
+    if (_rParams->GetName() == DVRParams::GetClassType())
+        return 100 * sqrtf(opacity);
+    else
+        return 100 * opacity;
+}
+
+float TFWidget::convertSliderValueToOpacity(int value) const
+{
+    if (_rParams->GetName() == DVRParams::GetClassType())
```

Comment:
Why does this only apply to the DVR renderer?

Reply:
The opacity of a volume rendering is proportional to the integrated opacity along the ray. This equals the opacity squared. The reason the old opacity slider felt unnatural (no change at higher slider values and too rapid change at lower slider values) is because we were essentially mapping the square root of the slider to the resulting opacity. If we apply the new mapping to other renderers which do not integrate opacity, we will be mapping the slider squared to opacity which would make every other renderer feel wrong.

The proper OOP solution would be for a renderer to have an `IntegratesOpacity` property or something similar however the DVR is the only renderer I could think of that would need it so I just opted for this.

On a side note, we may want to consider implementing this for the TF opacity control points too, although this will create the issue that we can no longer linearly interpolate them and will require much more effort than the single slider.

Reply:
How about simply adding a property to the TFWidget class that can be set by the user of the class, in this case DVRSubtabs.h?